gammapy/modeling/models/tests/test_spatial.py

```diff
@@ -103,6 +105,30 @@ def test_sky_gaussian():
     assert isinstance(model.to_region(), EllipseSkyRegion)
 
 
+def test_generalized_gaussian():
+    reval = 6
+    dr = 0.01
+    geom = WcsGeom.create(
+        skydir=(0, 0), binsz=dr, width=(2 * reval, 2 * reval), frame="galactic",
+    )
+    # check normalization is robut for a large set of values
+    for eta in np.arange(0.1, 1.01, 0.3):
```

Comment:
This kind of testing is usually implement using `purest.mark.parametrize()`, see e.g. https://github.com/gammapy/gammapy/blob/master/gammapy/maps/tests/test_base.py#L29. The big advantage is that the "parameterised" tests will be collected and executed by `pytest` as individual tests. So developers will get a much more meaningful test output.